catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageUtils.java

```diff
@@ -0,0 +1,279 @@
+    }
+
+    public static List<ExtrinsicObjectType> getExtrinsicObjects(
+            RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, ExtrinsicObjectType.class);
+    }
+
+    public static List<ExtrinsicObjectType> getExtrinsicObjects(
+            RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, ExtrinsicObjectType.class);
+    }
+
+    public static List<OrganizationType> getOrganizations(RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, OrganizationType.class);
+    }
+
+    public static List<OrganizationType> getOrganizations(
+            RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, OrganizationType.class);
+    }
+
+    public static List<PersonType> getPersons(RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, PersonType.class);
+    }
+
+    public static List<PersonType> getPersons(RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, PersonType.class);
+    }
+
+    public static List<AssociationType1> getAssociations(RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, AssociationType1.class);
+    }
+
+    public static List<AssociationType1> getAssociations(
+            RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, AssociationType1.class);
+    }
+
+    public static <T extends RegistryObjectType> List<T> getObjectsFromRegistryObjectList(
+            RegistryPackageType registryPackage, Class<T> type) {
+        List<T> registryObjects = new ArrayList<>();
+
+        if (registryPackage == null) {
+            return registryObjects;
+        }
+
+        if (registryPackage.isSetRegistryObjectList()) {
+            registryObjects =
+                    getObjectsFromRegistryObjectList(registryPackage.getRegistryObjectList(), type);
+        }
+
+        return registryObjects;
+    }
+
+    public static <T extends RegistryObjectType> List<T> getObjectsFromRegistryObjectList(
+            RegistryObjectListType registryObjectList, Class<T> type) {
+        List<T> registryObjects = new ArrayList<>();
+
+        if (registryObjectList == null) {
+            return registryObjects;
+        }
+
+        registryObjects.addAll(registryObjectList.getIdentifiable()
+                .stream()
+                .filter(identifiable -> type.isInstance(identifiable.getValue()))
+                .map(identifiable -> (T) identifiable.getValue())
+                .collect(Collectors.toList()));
+
+        return registryObjects;
+    }
+
+    public static SlotType1 getSlotByName(String name, List<SlotType1> slots) {
```

Comment:
Need to say that the return value can be `null` (use `@Nullable` annotation on the return type).